qutebrowser/mainwindow/tabwidget.py

```diff
@@ -112,6 +112,15 @@ def update_tab_title(self, idx):
         fields['index'] = idx + 1
         fields['id'] = widget.tab_id
         fields['title_sep'] = ' - ' if page_title else ''
+        y = widget.scroll_pos[1]
+        if y <= 0:
+            scroll_pos = 'top'
+        elif y >= 100:
+            scroll_pos = 'bot'
+        else:
+            scroll_pos = '{:2}%'.format(y)
+
+        fields['scroll_pos'] = '{}'.format(scroll_pos)
```

Comment:
This should be `str(scroll_pos)` instead (they do the same, but it's more idiomatic)